documentation/modules/auxiliary/admin/kerberos/inspect_ticket.md

```diff
@@ -0,0 +1,156 @@
+      Renew Till: 2032-11-18 13:52:00 +0000
+    Ticket:
+      Ticket Version Number: 5
+      Realm: WINDOMAIN.LOCAL
+      Server Name: cifs/dc.windomain.local
+      Encrypted Ticket Part:
+        Ticket etype: 18 (AES256)
+        Key Version Number: 2
+        Decrypted (with key: \x4b\x91\x2b\xe0\x36\x6a\x6f\x37\xf4\xa7\xd5\x71\xbe\xe1\x8b\x11\x73\xd9\x31\x95\xef\x76\xf8\xd1\xe3\xe8\x1e\xf6\x17\x2a\xb3\x26):
+          Times:
+            Auth time: 2022-11-21 13:52:00 UTC
+            Start time: 2022-11-21 13:52:00 UTC
+            End time: 2032-11-18 13:52:00 UTC
+            Renew Till: 2032-11-18 13:52:00 UTC
+          Client Addresses: 0
+          Transited: tr_type: 0, Contents: ""
+          Client Name: 'Administrator'
+          Client Realm: 'WINDOMAIN.LOCAL'
+          Ticket etype: 18 (AES256)
+          Encryption Key: 3436643936633032656264663030393931323461366635653364393932613763
+          Flags: 0x50a00000 (FORWARDABLE, PROXIABLE, RENEWABLE, PRE_AUTHENT)
+          PAC:
+            Validation Info:
+              Logon Time: 2022-11-21 13:52:00 +0000
+              Logoff Time: Never Expires (inf)
+              Kick Off Time: Never Expires (inf)
+              Password Last Set: No Time Set (0)
+              Password Can Change: No Time Set (0)
+              Password Must Change: Never Expires (inf)
+              Logon Count: 0
+              Bad Password Count: 0
+              User ID: 500
+              Primary Group ID: 513
+              User Flags: 0
+              User Session Key: \x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00
+              User Account Control: 528
+              Sub Auth Status: 0
+              Last Successful Interactive Logon: No Time Set (0)
+              Last Failed Interactive Logon: No Time Set (0)
+              Failed Interactive Logon Count: 0
+              SID Count: 0
+              Resource Group Count: 0
+              Group Count: 5
+              Group IDs:
+                Relative ID: 513, Attributes: 7
+                Relative ID: 512, Attributes: 7
+                Relative ID: 520, Attributes: 7
+                Relative ID: 518, Attributes: 7
+                Relative ID: 519, Attributes: 7
+              Logon Domain ID: S-1-5-21-3541430928-2051711210-1391384369
+              Effective Name: 'Administrator'
+              Full Name: ''
+              Logon Script: ''
+              Profile Path: ''
+              Home Directory: ''
+              Home Directory Drive: ''
+              Logon Server: ''
+              Logon Domain Name: 'WINDOMAIN.LOCAL'
+            Client Info:
+              Name: 'Administrator'
+              Client ID: 2022-11-21 13:52:00 +0000
+            Pac Server Checksum:
+              Signature: \x04\xe5\xab\x06\x1c\x7a\x90\x9a\x26\xb1\x22\xc2
+            Pac Privilege Server Checksum:
+              Signature: \x71\x0b\xb1\x83\x85\x82\x57\xf4\x10\x21\xbd\x7e
+```
+**Without Key**
```

Comment:
Can we put without key first - since that's the scenario most folk will actually have 😄

Reply:
sorted